docs/raiden_services.rst

```diff
@@ -0,0 +1,75 @@
+Raiden Services
+###############
+
+.. toctree::
+  :maxdepth: 3
+
+Introduction
+============
+
+The Raiden network includes auxiliary services, that enhance the overall user experience. These services include
+
+- finding short or cheap routes through the network and
+- allowing users to go offline safely by watching open payment channels.
```

Comment:
Should we maybe use the term `monitoring` instead of `watching`, since that's what we actually call the service?